pkg/features/kube_features.go

```diff
@@ -739,6 +739,15 @@ const (
 	// Enables In-Place Pod Vertical Scaling
 	InPlacePodVerticalScaling featuregate.Feature = "InPlacePodVerticalScaling"
 
+	// owner: @tallclair @esotsal
+	// alpha: v1.32
+	//
+	// Allow resource resize for containers in Guaranteed pods with integer CPU requests ( default false ).
+	// Applies only in nodes with InPlacePodVerticalScaling and CPU Manager features enabled, and
+	// CPU Manager Static Policy option set.
+	// Remove after InPlacePodVerticalScaling is GA based on user feedback that should be reflected in #1287 update.
```

Comment:
I think delete this line. Removal after GA with sufficient user feedback is assumed for all feature gate removal.

```suggestion
```

Reply:
Done